containers/java-postgres/README.md

```diff
@@ -0,0 +1,151 @@
+
+```yaml
+build:
+  context: ..
+  dockerfile: .devcontainer/Dockerfile
+    args:
+      # Update 'VARIANT' to pick an version of Java: 11, 17.
+      # Append -bullseye or -buster to pin to an OS version.
+      # Use -bullseye variants on local arm64/Apple Silicon.
+      VARIANT: 11-bullseye
+```
+
+You also can connect to PostgreSQL from an external tool when using VS Code by updating `.devcontainer/devcontainer.json` as follows:
+
+```json
+"forwardPorts": [ "5432" ]
+```
+
+Once the PostgreSQL container has port forwarding enabled, it will be accessible from the Host machine at `localhost:5432`. The [PostgreSQL Documentation](https://www.postgresql.org/docs/14/index.html) has:
+
+1. [An Installation Guide for PSQL], a CLI tool to work with a PostgreSQL database.
+2. [Tips on populating data](https://www.postgresql.org/docs/14/populate.html) in the database. 
+
+### Adding another service
+
+You can add other services to your `docker-compose.yml` file [as described in Docker's documentation](https://docs.docker.com/compose/compose-file/#service-configuration-reference). However, if you want anything running in this service to be available in the container on localhost, or want to forward the service locally, be sure to add this line to the service config:
+
+```yaml
+# Runs the service on the same network as the database container, allows "forwardPorts" in devcontainer.json function.
+network_mode: service:db
+```
+
+### Debug Configuration
+
+Note that only the integrated terminal is supported by the Remote - Containers extension. You may need to modify `launch.json` configurations to include the following value if an external console is used.
+
+```json
+"console": "integratedTerminal"
+```
+
+### Installing Maven or Gradle
+
+You can opt to install a version of Maven or Gradle by adding `INSTALL_MAVEN: "true"` or `INSTALL_GRADLE: "true"` to build args in `.devcontainer/docker-compose.yml`. Both of these are set by default. For example:
+
+```yaml
+args:
+  VARIANT: 11
+  INSTALL_GRADLE: "true"
+  INSTALL_MAVEN: "true"
+```
+
+Remove the appropriate arg or set its value to `"false"` to skip installing the specified tool.
+
+You can also specify the version of Gradle or Maven if needed.
+
+```yaml
+args:
+  VARIANT: 11
+  INSTALL_GRADLE: "true"
+  MAVEN_VERSION: "3.8.3"
+  INSTALL_MAVEN: "true"
+  GRADLE_VERSION: "7.2"
+```
+
+### Installing Node.js
+
+Given JavaScript front-end web client code written for use in conjunction with a Java back-end often requires the use of Node.js-based utilities to build, this container also includes `nvm` so that you can easily install Node.js. You can enable installation and change the version of Node.js installed or disable its installation by updating the `args` property in `.devcontainer/docker-compose.yml`.
+
+```yaml
+args:
+  VARIANT: 11
+  NODE_VERSION: "10" # Set to "none" to skip Node.js installation
```

Comment:
qq: does *NODE_VERSION* accept specific versions of node? Also does it accept the keyword latest to know when to get the latest version? If yes, may be worth to specify which versions, or that it accepts other args other than version number and "none".

If this is not relevant, or if it's already specified elsewhere in the docs, feel free to ignore or link the other content.

Reply:
hi @Kpopovs1, well, this MD section showing `NODE_VERSION: "10"` was left to match the 'main' java container version -- all that changed was the location of the arg, which moved to _docker-compose.yml_ from _devcontainer.json_
and yes, one can specify "NODE_VERSION": "lts/*" to 'get latest'

Reply:
upon further reflection, I updated the comment in the README.md, so how to use 'latest' is more clear